tests/integration/bless/main.go

```diff
@@ -51,6 +51,7 @@ const (
 var defaultStdoutPrefixes = []string{
 	"GAS USED:",
 	"STORAGE DELTA:",
+	"STORAGE FEE:",
 	"TOTAL TX COST:",
 	"EVENTS:",
 }
```

Comment:
The array size isn't large so it's not a problem, but using map seems like it could help avoid the loop in the `isValidStdoutLine` function.

Reply:
[526671d](https://github.com/gnoswap-labs/gnoswap/pull/1016/commits/526671d322f9c61a7634a2bd1314d74dbab20799)